client/lang/src/en.json

```diff
@@ -0,0 +1,11 @@
+{
+  "MFA.FIND_OUT_MORE": "Find out more",
```

Comment:
I mentioned this on Guy's PR the other day - I think we should make this root level called MultiFactorAuthentication or MultiFactorApp or something. While it's unlikely to have global collisions with other modules, you never know. It's only three letters.

Reply:
I've adjusted these. We'll want to pull all the other translations into this file at some point soon.